_development/configuration.md

```diff
@@ -72,6 +72,8 @@ This defines which motherboard you used for your 3D printer. It tells Marlin to
 
 Replace the `#define BOARD_RAMPS_14_EFB` line in Configuration.h with the line below that matches your current board.
 Check the `boards.h` file for the most up-to-date listing of supported boards, if you do not see yours listed here.
+NOTE: You MUST prefix the Board name with "MOTHERBOARD" in other words if you have done this correctly the line will read:
```

Comment:
That would be true except that the class declaration wasn't originally there for the default value. If we fix it so that it is there than I agree that is superfluous.